packages/requests/src/lib/requests-result.spec.ts

```diff
@@ -434,6 +438,174 @@ describe('requests result', () => {
     `);
   });
 
+  it('should cache response data', () => {
+    jest.useFakeTimers().setSystemTime(new Date('2020-01-01'));
```

Comment:
Why do you need this line?

Reply:
I copy pasted form the "should work" test

Reply:
I guess it's to avoid the stale time